### DISCUSSION TITLE
New translation in Italian (Traduzione in Italiano)

### DISCUSSION THREAD

**Discussion by gionatanfazio** [sircharlo/meeting-media-manager#192] [Translations]
Hello .. is there a planned work to translate this great app in Italian?

thanks 
Gionatan Fazio

**Comment by sircharlo** [comment]
Thanks for reaching out @gionatanfazio!
If you're comfortable forking a GitHub repo, feel free to do so and submit a PR with a `it.json` file added to the locales folder, with the contents copied from `en.json` and translated into your language.
If not, feel free to simply download [the english locale file](https://github.com/sircharlo/jw-meeting-media-fetcher/blob/master/locales/en.json) from [here](https://raw.githubusercontent.com/sircharlo/jw-meeting-media-fetcher/master/locales/en.json), translate the strings, and save and upload your translation in this discussion.
Thanks for your willing spirit!

**Reply by gionatanfazio** [reply]
Proposal of translation.
See file attached.

[it.json.txt](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8035084/it.json.txt)

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/ea833f77f7c53b780fffd146a965cead6cc83ffe. Thanks!!
